opsdroid/configuration/schema.yaml

```diff
@@ -0,0 +1,118 @@
+welcome-message: bool(required=False)
+
+connectors:
+  list(include('websocket', required=False),
+  include('shell', required=False),
+  include('ciscospark',required=False),
+  include('facebook',required=False),
+  include('github',required=False),
+  include('matrix',required=False),
+  include('slack',required=False),
+  include('telegram',required=False),
+  include('twitter',required=False))
+
+databases:
+  list(include('mongo', required=False),
+  include('redis', required=False),
+  include('sqlite', required=False),required=False)
+
+skills: list(include('skill',required=False))
```

Comment:
Same thing as the connectors, if there are no skills set up opsdroid will exit with code 1.
We need to have at least one skill and I'd go with the base `hello` skill since it's a good way for you to see if the bot is working as expected 😄

Reply:
now in the scheme the presence of skills is required !

Reply:
Great thanks for this 👍